src/views/forms/NewMarketForm/NewMarketSelectionStep.tsx

```diff
@@ -0,0 +1,463 @@
+                key: 'reference-price',
+                label: stringGetter({ key: STRING_KEYS.REFERENCE_PRICE }),
+                tooltip: 'reference-price',
+                value: (
+                  <Output
+                    type={OutputType.Fiat}
+                    value={assetToAdd.referencePrice}
+                    fractionDigits={tickSizeDecimal}
+                  />
+                ),
+              },
+              assetToAdd && {
+                key: 'message-details',
+                label: stringGetter({ key: STRING_KEYS.MESSAGE_DETAILS }),
+                value: (
+                  <Button
+                    action={ButtonAction.Navigation}
+                    size={ButtonSize.Small}
+                    onClick={() =>
+                      dispatch(
+                        openDialog({
+                          type: DialogTypes.NewMarketMessageDetails,
+                          dialogProps: { assetData: assetToAdd, clobPairId, liquidityTier },
+                        })
+                      )
+                    }
+                  >
+                    {stringGetter({ key: STRING_KEYS.VIEW_DETAILS })} →
+                  </Button>
+                ),
+              },
+              {
+                key: 'dydx-required',
+                label: (
+                  <span>
+                    {stringGetter({ key: STRING_KEYS.REQUIRED_BALANCE })}{' '}
+                    <Tag>{chainTokenLabel}</Tag>
+                  </span>
+                ),
+                value: (
+                  <Styled.Disclaimer>
+                    {stringGetter({
+                      key: STRING_KEYS.OR_MORE,
+                      params: {
+                        NUMBER: (
+                          <Styled.Output
+                            useGrouping
+                            type={OutputType.Number}
+                            value={initialDepositAmountBN}
+                            fractionDigits={initialDepositAmountDecimals}
+                          />
+                        ),
+                      },
+                    })}
+                  </Styled.Disclaimer>
+                ),
+              },
+            ].filter(isTruthy)}
+          />
+        }
+      >
+        {isDisconnected ? (
+          <OnboardingTriggerButton />
+        ) : (
+          <Button
+            type={ButtonType.Submit}
+            state={{ isDisabled: shouldDisableConfirmButton }}
+            action={ButtonAction.Primary}
+          >
+            {canModifyLiqTier
+              ? stringGetter({ key: STRING_KEYS.SAVE })
+              : stringGetter({ key: STRING_KEYS.PREVIEW_MARKET_PROPOSAL })}
+          </Button>
+        )}
+      </WithReceipt>
+    </Styled.Form>
+  );
+};
+
+const Styled: Record<string, AnyStyledComponent> = {};
+
+Styled.Form = styled.form`
+  ${formMixins.transfersForm}
+  ${layoutMixins.stickyArea0}
+  --stickyArea0-background: transparent;
+
+  h2 {
```

Comment:
can we match margin?
<img width="548" alt="image" src="https://github.com/dydxprotocol/v4-web/assets/9400120/dd8fa442-1d91-4756-9be1-992e21e0142b">